modulemd/meson.build

```
@@ -419,15 +385,6 @@ clang_args = [ '-i' ]
 test('clang_format', clang_format,
      args : clang_args + clang_files)
```

```suggestion
     args : clang_format_args + clang_files)
```
Completely optional: it would be more consistent with the style of the rest of this file if `clang_args` was renamed to `clang_format_args` here and two lines above--but there's no good reason to change it.

---

I'm not too worried about that, but thanks for the thorough review!